Mathmagician/Fibonacci.cs

```diff
@@ -4,9 +4,51 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
 namespace Mathmagician
 {
     public class Fibonacci
     {
+        
+        public static int IsFibonacci(int n)
+        {
+            int a = 0;
+            int b = 1;
+            // In N steps compute Fibonacci sequence iteratively.
+            for (int i = 0; i < n; i++)
```

Comment:
If you wanted 20 numbers of fibonacci to run, how many times does this loop run? I'm either misreading this or it would run a couple hundred times. There's probably a more efficient way for this to work. However, since this does work, I'm not going to require you to refactor it before I thumbs up. Just something to keep in mind if you want to go back and tweak for efficiency.